src/build.d

```diff
@@ -513,6 +514,25 @@ alias checkwhitespace = memoize!(function()
     return new DependencyRef(dep);
 });
 
+alias zip = memoize!(function()
+{
+    Dependency dep;
+    with (dep)
+    {
+        name = "zip";
+        target = "dmdsrc.zip";
```

Comment:
Absolute path?

Reply:
Yeah, based on the Makefile I was thinking it was generated the zip file in the "working directory", but that's not the case.